plugins/modules/global_parameter.py

```diff
@@ -111,14 +111,18 @@
 
 
 class ForemanCommonParameterModule(ForemanEntityAnsibleModule):
-    pass
+    def remove_sensitive_fields(self, entity):
+        if entity and 'hidden_value?' in entity:
+            entity['hidden_value'] = entity.pop('hidden_value?')
+        return super(ForemanCommonParameterModule, self).remove_sensitive_fields(entity)
```

Comment:
Should we remove value if `hidden_value` is `True`?

Reply:
I think that's reasonable, yes